rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/util/CovidRDTJsonFormUtils.java

```diff
@@ -91,6 +102,20 @@ protected void prePopulateRDTPatientFields(Patient patient, JSONObject field) th
         } else if (PATIENT_AGE.equals(key)) {
             field.put(JsonFormUtils.VALUE, patient.getAge());
         }
+
+        // pre-populate the patient detail name and dob
+        if (CovidConstants.FormFields.PATIENT_DETAIL.equals(field.getString(KEY))) {
```

Comment:
After making this change https://github.com/onaio/rdt-standard/pull/456/files#r476344796, the field keys can be referenced directly. No need for the `if` block.

Reply:
condition updated.